cursorless-talon/src/spoken_forms.py

```diff
@@ -18,58 +18,68 @@ def watch_file(spoken_forms: dict, filename: str) -> Callable:
     )
 
 
+P = ParamSpec("P")
+R = TypeVar("R")
+
+
+def get_defaults_from_file(
```

Comment:
Why not just call this function `watch_csv` and just internally use `init_csv_and_watch_changes` instead of the callable argument?

Reply:
I wanted the `watch_csv` function to mirror the kwargs from `init_csv_and_watch_changes`, excluding `default_values`.  If you know of a way to do that without this wrapper function then I am all for it 😅

Reply:
I just fixed a bug I introduced during some refactoring so pull if you're messing with it

Reply:
That part is fine. I'm not quite sure what passing the argument `init_csv_and_watch_changes` in gives us?

Reply:
If it has to be this complicated I'm still fine with it.

Reply:
The only reason to have the argument is that I can destructure the type in order to strip off the first two args from `init_csv_and_watch_changes`. I don't know any other way to do that

Reply:
I would probably just have copied the arguments personally instead of all this type stuff, but I think the result is really nice.

Reply:
Ha yeah thought about that 😄. Maybe that's a better solution? We still should get a type error if we mess something up so I don't think the duplication has many drawbacks. I'm worried this code is quite hard to read

Reply:
The advantage of this approach I guess is it mirrors the docstring and we see all the kwargs. Maybe I just add a docstring to the wrapper

Reply:
Go with either direction. I'm happy